pyEpiabm/pyEpiabm/host_progression_sweep.py

```diff
@@ -0,0 +1,50 @@
+import random
+from .abstract_sweep import AbstractSweep
+from .infection_status import InfectionStatus
+
+
+class HostProgressionSweep(AbstractSweep):
+    '''Class for sweeping through population and updating host infection status
+    and time to next infection status change.
+    '''
+
+    def _update_time_to_status_change(self):
+        """Method that assigns time until next infection status update,
+         given as a random integer between 1 and 10.
+        """
```

Comment:
```suggestion
        
        :return: Time until next infection status update
        :rtype: int
        """
```

Reply:
@lukedtc are you sure this is resolved? Think this change should be committed but let me know if you have any issues with it!